raiden/channel.py

```diff
@@ -283,8 +283,14 @@ def __init__(self, participant_address, participant_balance, get_block_number):
         # 0 is used in the netting contract to represent the lack of a
         # transfer, so this value must start at 1
         if isinstance(get_block_number, int):
+            # if get_block_number == 0:
```

Comment:
Please don't leave commented code inside a commit. Amend the commit and remove it.